server/config/passport.js

```diff
@@ -45,12 +45,12 @@ passport.use(
           done(null, false, { msg: accountSuspensionMessage });
           return;
         }
-        user.comparePassword(password, (innerErr, isMatch) => {
+        user.comparePassword(password).then((isMatch) => {
```

Comment:
Wr can use async await syntax here also.

Reply:
For sure.  I figured I’d convert the passport file in a separate PR.  So I made the minimum change to keep it working for now.